c/src/ml-api-service-remote.c

```diff
@@ -756,3 +754,41 @@ ml_service_remote_register (ml_service_h handle, ml_option_h option, void *data,
     nns_edge_data_destroy (data_h);
   return ret;
 }
+
+/**
+ * @brief Sets the information in ml-service remote handle.
+ */
+int ml_service_remote_set_information (ml_service_h handle, const char *name, void *value, ml_data_destroy_cb destroy)
+{
+  ml_service_s *mls = (ml_service_s *) handle;
+  _ml_remote_service_s *remote_s = NULL;
+  int status;
+
+  check_feature_state (ML_FEATURE_SERVICE);
+
+  if (!_ml_service_handle_is_valid (mls)) {
+    _ml_error_report_return (ML_ERROR_INVALID_PARAMETER,
+        "The parameter, 'handle' (ml_service_h), is invalid. It should be a valid ml_service_h instance.");
+  }
+
+  if (!STR_IS_VALID (name))
+    _ml_error_report_return (ML_ERROR_INVALID_PARAMETER,
+        "The parameter, name '%s', is invalid.", name);
+
+  if (!value)
+    _ml_error_report_return (ML_ERROR_INVALID_PARAMETER,
+        "The parameter, value, is NULL. It should be a valid pointer.");
+
+  remote_s = (_ml_remote_service_s *) mls->priv;
+
+  status = ml_option_set (remote_s->info, name, value, destroy);
+  if (status != ML_ERROR_NONE) {
+    _ml_error_report_return (status, "Failed to set the information in ml-service remote handle.");
+  }
+
+  if (g_ascii_strcasecmp (name, "path") == 0) {
+    status = ml_service_remote_set_path (mls);
```

Comment:
This function only sets path now. Is there any plan to support other names in the future?
I think it would be better make error or log to this part if we don't have any plan to support other options.

Also although this is not for this PR, it seems necessary to write a test case for `ml_service_(remote_)set_information`

Reply:
Thanks. `ml_service_set_information()` was added as you suggested by @jaeyun-jung.
All general information is set by this method, but the only one property of ml-service-remote changed by this method is `path`. Other information has no effect on the ml-service.
If there is additional information that needs to be set in ml-service property, it is easy to modify because it only needs to modify the internal implementation without changing the API.
So, If necessary, let's change it later!